python/ray/data/tests/test_groupby_e2e.py

```diff
@@ -110,6 +110,60 @@ def test_map_groups_with_gpus(
     assert rows == [{"id": 0}]
 
 
+def test_groupby_with_column_expression(
+    ray_start_regular_shared_2_cpus,
+    configure_shuffle_method,
+    disable_fallback_to_object_extension,
+    target_max_block_size_infinite_or_default,
```

Comment:
I know we do this for the other tests in this file, but I don't think we need `target_max_block_size_infinite_or_default` here